src/Helmet.js

```diff
@@ -287,15 +313,20 @@ const generateTagsAsString = (type, tags) => {
     return stringifiedMarkup;
 };
 
-const generateTitleAsReactComponent = (type, title) => {
+const generateTitleAsReactComponent = (type, title, attributes) => {
     // assigning into an array to define toString function on it
+    const props = {
+        key: title,
+        [HELMET_ATTRIBUTE]: true
+    };
+    Object.keys(attributes).forEach((attribute) => {
+        props[attribute] = attributes[attribute];
+    });
+
```

Comment:
You would do a conversion here using `REACT_TAG_MAP` to convert `itemprop` to `itemProp`